src/modm/board/disco_f469ni/board.xml

```diff
@@ -9,7 +9,7 @@
     <option name="modm:target">stm32f469nih</option>
     <option name="modm:platform:uart:3:buffer.tx">2048</option>
     <option name="modm:platform:cortex-m:allocator">tlsf</option>
-    <option name="modm:tlsf:minimum_pool_size">16777216</option>
+    <option name="modm:tlsf:minimum_pool_size">2**24</option>
```

Comment:
I'm not sure that this is much more readable to be honest. Something like `16MiB` would be better.

Reply:
Agreed, SI prefix (plus binary prefix) support would be better!
Maybe we can "just" do a replace on the input string? Like `input.replace("Mi", "*2**20")`? and then for every prefix we want?

Reply:
Removed this for now, I think there needs to be a better solution.